tests/test_compilesim.py

```diff
@@ -902,13 +914,13 @@ def compareIO(self, sim_trace_a, expected_output):
         sim_trace_a.print_trace(output, compact=True)
         self.assertEqual(output.getvalue(), expected_output)
 
-    def test_function_RomBlock(self):
+    def romBlock_test_helper(self, bidwidth):
```

Comment:
I know you didn't start this, but we should try to avoid mixing capitalization styles. So this should be called `rom_block_test_helper`. There are a few more `romBlock`s in this change that should be `rom_block`s. Also there's a typo, `bidwith` should be `bitwidth` (happens again a few more times below)